doc/source/manual/xtrabackup_sst.rst

```diff
@@ -49,208 +47,203 @@ Xbstream is highly recommended. Refer to :ref:`Xbstream v/s Tar <tar_ag_xbstream
+Used to enable rebuilding of index on joiner node. Set to ``1`` to enable. This is independent of compaction, though compaction enables it. Rebuild of     
+indexes may be used as an optimization.
+
+.. note:: :bug:`1192834` affects this option.
+
 .. option:: time
 
-    :Values: 0,1  
+    :Values: 0, 1 
     :Default: 0   
 
-Enabling it instruments key stages of backup/restore in SST.
+Enabling this option instruments key stages of backup and restore in SST.
                
-.. option:: rlimit 
+.. option:: rlimit
 
-    :Values: x(k|m|g|t) 
-    
-Ratelimit to ``x`` kilobytes, megabytes etc. Refer to `pv(1) <http://linux.die.net/man/1/pv>`_ for details. Note this rate-limiting happens on donor. The rationale behind this is to not allow SST to saturate the donor's regular cluster operations and/or to ratelimit for other purposes.
+    :Example: rlimit=128k
+
+Used to set a a ratelimit in bytes. Add a suffix (k, m, g, t) to specify other units. For example, ``128k`` is 128 kilobytes. Refer to `pv(1) <http://linux.die.net/man/1/pv>`_ for details.
+
+.. note:: Rate is limited on donor node. The rationale behind this is to not allow SST to saturate the donor's regular cluster operations or to limit the rate for other purposes.
 
 .. option:: incremental
 
-    :Values: 0,1
+    :Values: 0, 1
     :Default: 0
 
-To be set on joiner only, supersedes IST if set. Currently requires
-manual setup. Hence, not supported currently.
+Used to sepersede IST on joiner node. Requires manual setup and is not supported currently.
 
 .. option:: use_extra
 
-    :Values: 0,1
+    :Values: 0, 1
     :Default: 0
 
 
-If set to 1, SST will use the thread pool's `extra_port <http://www.percona.com/doc/percona-server/5.6/performance/threadpool.html#extra_port>`_. Make sure that thread pool is enabled and extra_port option is set in my.cnf before you turn on this option.
+Used to force SST to use the thread pool's `extra_port <http://www.percona.com/doc/percona-server/5.6/performance/threadpool.html#extra_port>`_. Make sure that thread pool is enabled and the ``extra_port`` option is set in :file:`my.cnf` before you enable this option.
 
 .. option:: cpat
 
-During the SST, the :term:`datadir` is cleaned up so that state of other node can be restored cleanly. This option provides the ability to define the files that need to be deleted before the SST. It can be set like: :: 
+Used to define the files that need to be deleted in the :term:`datadir` before running SST, so that the state of the other node can be restored cleanly. For example: :: 
 
   [sst]
   cpat='.*galera\.cache$\|.*sst_in_progress$\|.*grastate\.dat$\|.*\.err$\|.*\.log$\|.*RPM_UPGRADE_MARKER$\|.*RPM_UPGRADE_HISTORY$\|.*\.xyz$'
 
-**NOTE:** This option can only be used when :variable:`wsrep_sst_method` is set to xtrabackup-v2.
+.. note:: This option can only be used when :variable:`wsrep_sst_method` is set to ``xtrabackup-v2``.
 
 .. option:: sst_special_dirs
    
-     :Values: 0,1
+     :Values: 0, 1
```

Comment:
Not sure if I understood this correctly.
Is this like "Only if this value is set to 1 then innodb_data/log_home_dir will be parsed ?
Let me put a marker here. We need to try this out to understand it exactly.